Library/Formula/consul.rb

```diff
@@ -13,6 +13,8 @@ class Consul < Formula
     sha256 "67052157266d536598ba7eb5af31a10fd3e609ca812be6d50869d0fe80a583f9" => :mountain_lion
   end
 
+  option "with-web-ui", "Installs the consul web ui (to /usr/local/share/consul directory)"
```

Comment:
`/usr/local/share/consul `

This won't always be true. Although we encourage the use of `/usr/local` & our bottles are built for there you can actually stick Homebrew in any directory.